docs/UserGuide.md

```diff
@@ -147,6 +156,19 @@ Clears all entries from the address book.
 
 Format: `clear`
 
+### Viewing a specific contact : `view`
+
+Views all the contact information of the specified contact.
+
+Format: `view t/TELEGRAM_HANDLE`
+
+* Displays all contact information of the person with specified `TELEGRAM_HANDLE`
+* `TELEGRAM_HANDLE` must be more than 4 characters, and can only contain letters, numbers and underscores.
+
+Examples:
+* `view` followed by `t/bob12` displays page containing all the information of the person with telegram handle `@bob12`<br>
```

Comment:
do you think `view t/bob12` is easier for the reader to understand? cus on first glance a reader might think they have to enter `view` then enter `t/bob12` again in the current example

Reply:
this follows the convention for all the existing commands in the UG, so if wanna change may need to change for everything else too